docker/compose.yml

```diff
@@ -0,0 +1,16 @@
+name: adamant_example
+services:
+    adamant_example:
+        container_name: ${DOCKER_CONTAINER_NAME}
+        volumes:
+            - type: bind
+              source: ../../adamant
```

Comment:
while this is a decent assumption, we should allow users to specify where adamant lives if it is elsewhere. So I am thinking best to pull this path from `docker_config.sh`

Reply:
Addressed in [1752748](https://github.com/lasp/adamant_example/pull/25/commits/17527488b8ea945ca8c836aae4ddc2dafcf49156).